.eslintrc.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "vazco/typescript",
   "parser": "@typescript-eslint/parser",
+  "plugins": ["prettier"],
```

Comment:
`vazco/typescript` already includes that as well as the `prettier/prettier` rule, so it could be removed.